gef-ui/server/services/__tests__/api.test.js

```diff
@@ -367,3 +367,24 @@ describe('getAddressesByPostcode()', () => {
     await expect(api.getAddressesByPostcode({ postcode: 'invalid', userToken })).rejects.toThrowError('Invalid postcode');
   });
 });
+
+describe('getAmendment()', () => {
+  it(`should return ${HttpStatusCode.Ok} when a facility can be found`, async () => {
+    Axios.get.mockReturnValue(Promise.resolve({ data: { status: HttpStatusCode.Ok } }));
+    const response = await api.getAmendment({ facilityId: validMongoId, amendmentId: validMongoId, userToken });
+    expect(response).toEqual({ status: HttpStatusCode.Ok });
+  });
+
+  it('should throw an error if there is an api error', async () => {
+    Axios.get.mockReturnValue(Promise.reject(new AxiosError()));
+    await expect(api.getAmendment({ facilityId: validMongoId, amendmentId: validMongoId, userToken })).rejects.toThrowError(AxiosError);
+  });
+
```

Comment:
legacy in this file so happy to leave but `toThrowError` is deprecated

Reply:
Changed, thanks!